test/docker_tests/test_docker_compose_scenarios.py

```
@@ -0,0 +1,440 @@
+        config_file.write_text("# Test configuration\n")
+
+    # Create basic db file
+    db_file = base_dir / "test_data" / "app_db" / "app.db"
+    if not db_file.exists():
+        # Create a minimal SQLite database
+        import sqlite3
+        conn = sqlite3.connect(str(db_file))
+        conn.close()
+
+
+def _run_docker_compose(compose_file: pathlib.Path, project_name: str, timeout: int = 30, env_vars: dict = None) -> subprocess.CompletedProcess:
```

Ruff (0.14.2)

Unused function argument: `env_vars`

(ARG001)